easybuild/easyconfigs/r/Ruby-FPM/Ruby-FPM-2.1.5-1.0.0-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-FPM'
```

Comment:
I'd suggest to make -FPM and versionsuffix and move this under `r/Ruby`, what would you think about this idea?
(considering the more generic case of different bundles based upon Ruby, Python, Perl, R etc...)